test/parallel/test-crypto-randomuuid.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const common = require('../common');
+
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const { randomUUID } = require('crypto');
+
+const last = [
+  '00000000-0000-0000-0000-000000000000'
+];
+
+// Generate a number of UUID's to make sure we're
+// not just generating the same value over and over
+// and to make sure the batching changes the random
+// bytes.
+for (let n = 0; n < 100; n++) {
```

Comment:
Wouldn't you have to generate >128 (`const kBatchSize = 128;`)
 UUIDs in order to verify that the batching indeed changes (as the comment above suggests)?

Reply:
Good catch. Forgot to update that when bumping up the cache size.